<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="prev_screen_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Vissza az előző elemzésekhez</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="similar_word_textbox_1">
   <property name="geometry">
    <rect>
     <x>1560</x>
     <y>260</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="similar_words_button_1">
   <property name="geometry">
    <rect>
     <x>1560</x>
     <y>300</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Megtekintés</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="similar_word_textbox_2">
   <property name="geometry">
    <rect>
     <x>1560</x>
     <y>730</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>1180</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="similar_words_vbox_1"/>
  </widget>
  <widget class="QPushButton" name="similar_words_button_2">
   <property name="geometry">
    <rect>
     <x>1560</x>
     <y>770</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Megtekintés</string>
   </property>
  </widget>
  <widget class="QLabel" name="word_similarity_label_1">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>1000</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="word_similarity_label_2">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>1060</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="next_screen_button">
   <property name="geometry">
    <rect>
     <x>1750</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>További elemzések</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>530</y>
     <width>1180</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="similar_words_vbox_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
